server/app/controllers/api/ProgramApplicationsApiController.java

```diff
@@ -0,0 +1,25 @@
+package controllers.api;
+
+import auth.ProfileUtils;
+import com.google.inject.Inject;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import play.mvc.Http;
+import play.mvc.Result;
+
+/** API controller for admin access to a specific program's applications. */
+public final class ProgramApplicationsApiController extends CiviFormApiController {
+
+  @Inject
+  public ProgramApplicationsApiController(ProfileUtils profileUtils) {
+    super(profileUtils);
+  }
+
+  // Introduced to test API authentication logic. Implementation is next. Did not
```

Comment:
[nit] This would be a good place for a TODO w/ issue number

Reply:
done